source/designer/source/DocumentCanvas.cpp

```diff
@@ -67,7 +67,7 @@ void DocumentCanvas::SelectControls( ControlList& CtrlList )
 {
 	m_selectionLayer->ClearSelection();
 
-	for ( ControlList::List::const_iterator it = CtrlList.list.begin(); it != CtrlList.list.end(); ++it )
+	for ( ControlList::List::const_iterator it = CtrlList.list.cbegin(); it != CtrlList.list.cend(); ++it )
```

Comment:
Good attention to detail. These will probably all get changed to for-range over time.